lib/Backend/Lower.cpp

```diff
@@ -3833,16 +3835,18 @@ Lowerer::GenerateProfiledNewScObjArrayFastPath(IR::Instr *instr, Js::ArrayCallSi
     Func * func = this->m_func;
     IR::LabelInstr * helperLabel = IR::LabelInstr::New(Js::OpCode::Label, func, true);
     uint32 size = length;
+    bool containMissingItems = size != 0;
```

Comment:
Sure, I can. Any reason for having it in if-condition vs. having it like this?

---
In reply to: [71650529](https://github.com/Microsoft/ChakraCore/pull/1304#discussion_r71650529) [](ancestors = 71650529)